indico/modules/users/templates/personal_data.html

```diff
@@ -48,4 +63,29 @@
             </div>
         </div>
     </div>
+
+    <script>
+        $('.pic-form').on('indico:fieldsSaved', function(e, response) {
+            var hidden = !response.content;
```

Comment:
Let's use `const` (or `let` if necessary) and arrow functions for callbacks (at least when `this` is not needed, so not in this callback) in new code